src/common/table/Table.tsx

```diff
@@ -253,9 +253,12 @@ const Table = <D extends { id: string }>({
     gotoPage(0);
   }, [gotoPage, state.sortBy, state.filters, state.globalFilter]);
 
+  const initialSortBy = initialState?.sortBy;
+
   useEffect(() => {
+    if (initialSortBy?.[0]?.id === state.sortBy[0]?.id && initialSortBy?.[0]?.desc === state.sortBy[0]?.desc) return;
```

Comment:
This is a bit complicated, could this be expressed a bit more clearly?

Reply:
Done.